tests/envs/test_tafficjunction.py

```diff
@@ -0,0 +1,60 @@
+import gym
+import pytest
+import ma_gym
+from pytest_cases import pytest_parametrize_plus, fixture_ref
+
+
+@pytest.fixture(scope='module')
+def env():
+    env = gym.make('TrafficJunction-v0')
+    yield env
+    env.close()
+
+
+def test_init(env):
+    assert 1 <= env._n_max <= 10, f'N_max must be between 1 and 10, got {env._n_max}'
```

Comment:
`{env._n_max}` is not compatible with `python 3.5`. 

Let's just use:
```
assert 1 <= env._n_max <= 10, 'N_max must be between 1 and 10, got {}'.format(env._n_max)
```

Also, the issue is across the entire test file.

Reply:
Todo for me: I will add "3.5" constraints in the workflow.